src/scss/attendee/effects/sparkle.scss

```diff
@@ -1,6 +1,7 @@
 @use "../theme";
 
 $blur: 5px;
+$blur--medium: 14px;
```

Comment:
[OOS][NOTE] by adding `$blur--medium` we will have `$blur` meaning be... `light`? `normal`?

Maybe we can do it like `$blur--lg` for `14px` and `$blur--md` for the `5px` since that one is used a lot.

Reply:
done: https://github.com/sparkletown/sparkle/pull/3015/commits/42838ddbe607e71b4b4b587e9c83c1883992155b